torch/csrc/distributed/rpc/python_call.cpp

```diff
@@ -6,29 +6,31 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
-PythonCall::PythonCall(
-    std::vector<char> pickledPayload,
-    std::vector<torch::Tensor> tensors)
-    : pickledPayload_(std::move(pickledPayload)),
-      tensors_(std::move(tensors)) {}
+PythonCall::PythonCall(SerializedPyObj&& serializedPyObj)
+    : serializedPyObj_(std::move(serializedPyObj)) {}
 
 Message PythonCall::toMessage() && {
+  auto payload = std::vector<char>(
+      serializedPyObj_.payload_.begin(), serializedPyObj_.payload_.end());
```

Comment:
We are making a copy here? 
We should target this consolidation not introducing performance overhead, right?
Can we "move" from `serializedPyObj_ `? (seems need to first refactor SerializedPyObj to have vector<char> instead of string).